scania-digital-design-system/tegel
# test(header): Adding unit tests

**Describe pull-request**  
Unit tests for header component

**Solving issue**  
Fixes: [CDEP-3003](https://tegel.atlassian.net/browse/CDEP-3003)

**How to test**  
1. Checkout code
2. `npm run test` the core package to check that tests run


[CDEP-3003]: https://tegel.atlassian.net/browse/CDEP-3003?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
